src/main/resources/assets/api-docs.yaml

```diff
@@ -488,6 +488,36 @@ paths:
             application/json:
               schema:
                 $ref: './schemas/DataAccessRequest.yaml'
+  /api/dar/progress_report/{parentReferenceId}:
+    post:
+      summary: Create a new DAR progress report
+      description: Create a new DAR progress report
+      parameters:
+        - name: parentReferenceId
+          in: path
+          description: The referenceId of the Data Access Request
+          required: true
+          schema:
+            type: string
+        - name: dar
+          in: body
+          description: The fields that represent a Data Access Request, json format.
+          required: true
+          schema:
+            $ref: './schemas/DataAccessRequest.yaml'
```

Comment:
This is in an older version of swagger syntax, the current version needs this to be in the `requestBody` section.